UIComponents/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="paypal_blue">#60CDFF</color>
```

Comment:
Are these color resources used anywhere?

Related question - would it be better to have all of the color hex values in this colors.xml file?